src/components/PasswordForm/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { func } from 'prop-types';
+import { useStatus, LOADING, SUCCESS } from '@rootstrap/redux-tools';
+
+import { changePassword } from 'actions/userActions';
+import Input from 'components/common/Input';
+import useForm from 'hooks/useForm';
+import useValidation from 'hooks/useValidation';
+import passwordValidations from 'validations/passwordValidations';
+import ErrorView from 'components/common/ErrorView';
+import useTextInputProps from 'hooks/useTextInputProps';
+import strings from 'localization';
+import MainButton from 'components/common/MainButton';
```

Comment:
what do you think about re-ordering the imports, at least for me is easier read when they are in the same context, like components, hooks, validations, etc. 

```suggestion

import useForm from 'hooks/useForm';
import useValidation from 'hooks/useValidation';
import useTextInputProps from 'hooks/useTextInputProps';

import Input from 'components/common/Input';
import ErrorView from 'components/common/ErrorView';
import MainButton from 'components/common/MainButton';

import passwordValidations from 'validations/passwordValidations';

import strings from 'localization';
```